hedera-node/hedera-app/src/test/resources/bootstrap/network.json

```diff
@@ -127,37 +124,6 @@
       "adminKey": {
         "ed25519": "CqjiEGTGHquG4qnBZFZbTnqaQUYQbgps0DqMOVoRDpI="
       }
-    },
-    "tssEncryptionKey": "ASM="
-  }],
-  "tssMessages": [{
```

Comment:
Not a blocker, just curious if devops is dependent on these to be present in any network?

Reply:
No environment is depending on this fields as discussed